test/GitHub.InlineReviews.UnitTests/ViewModels/PullRequestReviewCommentViewModelTests.cs

```diff
@@ -172,12 +191,12 @@ public void IsDisabledWhenSessionHasPendingReview()
             }
 
             [Test]
-            public void IsDisabledWhenSessionHasNoPendingReview()
+            public void IsEnabledWhenSessionHasNoPendingReview()
             {
                 var session = CreateSession(false);
                 var target = CreateTarget(session);
 
-                Assert.That(target.StartReview.CanExecute(null), Is.False);
+                Assert.That(target.StartReview.CanExecute(null), Is.True);
```

Comment:
I don't think this is correct. This was asserting that you can't start a review when `Body` is empty. With this change the "Start review" button is enabled with no body:

![image](https://user-images.githubusercontent.com/1775141/43712265-e7ff5236-9975-11e8-9cde-d645101ab1b0.png)

The test should probably have been named `IsDisabledWhenSessionHasNoPendingReviewAndEmptyBody` to match the following `IsEnabledWhenSessionHasNoPendingReviewAndBodyNotEmpty`.

Reply:
Okay cool, those tests make more sense now.